### ISSUE TITLE
More accessibility with screen reader 

### ISSUE THREAD

**Issue by MesterPerfect** [hellzerg/optimizer#150]
There is an accessibility issue with Jaws/NVDA screen readers
In Universal tab and windows tab, the names of the elements cannot be recognized using a screen reader
While names of elements can be easily identified in all tabs
Can you fix that

**Comment by MesterPerfect** [comment]
@hellzerg

**Comment by hellzerg** [comment]
I am busy this week. I'll look into it when I find the time.

**Comment by hellzerg** [comment]
Which screen reader are you using and facing the problem?

**Comment by MesterPerfect** [comment]
> Which screen reader are you using and facing the problem?

 The problem is with all screen readers
jaws, NVDA, and Narrator.
The problem is that the label is not associated with the PictureBox in the universalTab
That is, if you click on the name of any element, the state of the button is not changed
The same button must be pressed to change its state

**Comment by MesterPerfect** [comment]
If clicked, the state of the item will not be changed

![1](https://user-images.githubusercontent.com/44323153/166342842-db500172-2d0b-443d-be42-56c92584cdae.jpg)

While here if it is clicked, the state of the item will change

![2](https://user-images.githubusercontent.com/44323153/166342869-a7391037-317f-44dd-8427-2c411e558cf6.jpg)

**Comment by hellzerg** [comment]
Yes, this functionality prevents false, accidental clicks.
Also, if you hover over the text of each toggle, a small tip explaining what the toggle does will pop.

**Comment by MesterPerfect** [comment]
> Yes, this functionality prevents false, accidental clicks. Also, if you hover over the text of each toggle, a small tip explaining what the toggle does will pop.

It may prevent accidental false clicks, but it also prevents a screen reader from recognizing names of elements, making it difficult for users of screen readers to know name of each element.

**Comment by hellzerg** [comment]
Will try to see what I can do.

**Comment by hellzerg** [comment]
You may try v12.9 for accessibility support in toggles. It should work.

**Comment by MesterPerfect** [comment]
> You may try v12.9 for accessibility support in toggles. It should work.
This is great, everything works fine.
